packages/jellyfish-api-core/src/category/blockchain.ts

```diff
@@ -367,3 +376,13 @@ export interface BlockStats {
   utxo_size_inc: number
   feerate_percentiles: [number, number, number, number, number]
 }
+
+export interface MempoolInfo {
+  loaded: boolean
+  size: number
+  bytes: number
+  usage: number
+  maxmempool: number
+  mempoolminfee: number
+  minrelaytxfee: number
```

Comment:
size, bytes, usage, maxmempool are int64 -> use bignumber
fee is amount -> use bignumber as well

Reply:
Change the precision to `bignumber`.